### PR TITLE
Expand collapse layers

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Part of https://github.com/CartoDB/cartodb/issues/7342.

expanding

@viddo 👍  or 👎  ? Thanks!

### CODE REVIEW

**Comment by alonsogarciapablo** [comment]
@piensaenpixel can you please verify this is pixel perfect? 😛

File: lib/assets/javascripts/cartodb3/data/layer-definition-model.js [hunk 1]
```diff
@@ -179,5 +179,18 @@ module.exports = Backbone.Model.extend({
     if (this.styleModel) {
       this.styleModel.resetStyles();
     }
+  },
+
+  getNumberOfAnalyses: function () {
+    var analysisNode = this.getAnalysisDefinitionNodeModel();
+    if (!analysisNode) {
```
Comment (viddo) [inline lib/assets/javascripts/cartodb3/data/layer-definition-model.js]: By the look of it if there is no source the while would skip due to the first condition (so would return count = 0), so is this guard really necessary?

File: lib/assets/javascripts/cartodb3/editor/layers/layer.tpl [hunk 2]
```diff
@@ -27,9 +27,16 @@
...
-    <button class="CDB-Text CDB-Size-small u-actionTextColor u-upperCase js-add-analysis" data-layer-id="<%- layerId %>">
-      <%- _t('editor.layers.layer.add-analysis') %>
-    </button>
+    <div class="Editor-ListLayer-subtitle">
+      <button class="CDB-Text CDB-Size-small u-actionTextColor u-upperCase js-add-analysis" data-layer-id="<%- layerId %>">
+        <%- _t('editor.layers.layer.add-analysis') %>
+      </button>
+      <% if (isCollapsed && numberOfAnalyses > 0) { %>
+        <p class="CDB-Text CDB-Size-small u-secondaryTextColor">
+          (<%- numberOfAnalyses %>) <%- numberOfAnalyses === 1 ? 'Analysis' : 'Analyses' %>
```
Comment (viddo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer.tpl]: This should be handled through [`_t() pluralization`](https://github.com/airbnb/polyglot.js#pluralization)

**Comment by viddo** [comment]
Some minor things, otherwise 👍

File: lib/assets/javascripts/cartodb3/data/layer-definition-model.js (hunk 1 above)
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/data/layer-definition-model.js]: Totally! I'll fix it. Thanks!

File: lib/assets/javascripts/cartodb3/editor/layers/layer.tpl (hunk 2 above)
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer.tpl]: :+1:

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 3]
```diff
@@ -77,11 +77,18 @@
   cursor: pointer;
 }
 
+.Editor-ListLayer-subtitle {
```
Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: .Editor-listLayerSubtitle

File: lib/assets/javascripts/cartodb3/editor/layers/layer.tpl [hunk 4]
```diff
@@ -27,9 +27,16 @@
         </li>
       </ul>
     </div>
-    <button class="CDB-Text CDB-Size-small u-actionTextColor u-upperCase js-add-analysis" data-layer-id="<%- layerId %>">
-      <%- _t('editor.layers.layer.add-analysis') %>
-    </button>
+    <div class="Editor-ListLayer-subtitle">
```
Comment (piensaenpixel) [inline lib/assets/javascripts/cartodb3/editor/layers/layer.tpl]: i think here we can use the helpers "u-flex" and "u-justifySpace" and if you want _petarlo_ "u-alignCenter" and remove Editor-ListLayer thing

**Comment by piensaenpixel** [comment]
👍  puto amo

**Pull request merged**